lib/core/src/input/input.tsx

```diff
@@ -56,61 +59,68 @@ export interface InputProps {
 	// Style
 
 	/**
-	 * Icon appear in an input. See the [Icons guide][1] to learn more.
+	 * Icon in the Input. See the [Icons guide][1] to learn more.
+	 *
 	 * [1]: /docs/guides-icons--primary
 	 */
 	icon?: IconComponent;
 	/**
-	 * Style of the text box. Choose one from `Input.styles`
+	 * Style of the text box. Choose one from `Input.styles`.
 	 */
 	style?: InputStyle;
 	/**
-	 * Size of the text box. Choose one from `Input.sizes`
+	 * Size of the text box. Choose one from `Input.sizes`.
 	 */
 	size?: InputSize;
 
 	// Attributes
 
 	/**
-	 * The element's identifier. See [MDN reference][1]
+	 * The [HTML `id`] attribute of the Input
+	 *
 	 * [1]: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id
 	 */
 	id?: string;
 	/**
-	 * The name of the input, used when submitting HTML form. See [MDN reference][1]
+	 * The [HTML `name`] attribute of the Input, usually used in forms
```

Comment:
(1) Why this sentence not punctuated at the end?

Reply:
Sorry, I should have a doc to outline these things (part of our contributing guide). This doesn't have a full stop because:
- It is meant to be an item inside a list, and
- It is not a complete sentence, since the "subject" and even the "verb" part is understood implicitly and omitted. If this is a sentence, it should be "This prop sets the HTML name attribute...".

See:
- https://style.ons.gov.uk/house-style/bullet-points/
- https://simple.wikipedia.org/wiki/Punctuation#Period_or_full_stop